spec/customer_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'customer'
+
+describe Customer do
+
+  let(:menu) {double :menu, show_menu:{cheese: 7,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.<br>Space missing after colon.<br>Space inside { missing.